styles/navbar.css

```diff
@@ -49,10 +52,11 @@
     right: -100%;
     width: 80%;
     height: 100vh;
-    padding: 2rem;
+    padding: 1rem 1rem 4rem 1rem;
```

Comment:
Define global variable for things like padding, margin and border-radius for consistency in design

Reply:
Thanks here, I did it manually since it was the only place till now for this padding combination. I will definitely set some variables if I find some duplicates.